pkg/vcluster-rancher-extension-ui/setupButtonInjector.js

```diff
@@ -23,7 +23,8 @@ export function setupButtonInjector() {
 
         copiedButton.addEventListener("click", (e) => {
           e.preventDefault();
-          window.location.href = "/vCluster/c/_/dashboard";
+          // This will break it locally, but works in prod. No way for me to determine if we're on a local env or prod.
+          window.location.href = "/dashboard/vCluster/c/_/dashboard";
```

Comment:
Perhaps we could look into injecting NODE_ENV into the page. It seems that vue already does this? https://cli.vuejs.org/guide/mode-and-env.html#using-env-variables-in-client-side-code

Not a blocker for the PR, but should make development easier for you